src/pages/Card/_components/DropDownTop.tsx

```diff
@@ -9,17 +9,33 @@ interface DropDownTopProps {
 }
 
 function DropDownTop({ title, onClick, isActive }: DropDownTopProps) {
+  // title을 조건에 따라 변경
+  const getDisplayTitle = (title: string) => {
+    switch (title) {
+      case 'HYUNDAI_ORIGINALS':
+        return 'Hyundai Originals';
+      case 'CHAMPION_BRANDS':
+        return 'Champion Brands';
+      case 'AFFILIATE':
+        return '제휴카드';
+      case 'MY_BUSINESS':
+        return 'My Business';
+      default:
+        return title;
+    }
+  };
```

Comment:
이 함수는 렌더링 될 때 다시 생성될 필요가 없으니, 컴포넌트 밖으로 빼도 좋을것 같아요~~
```suggestion
const getDisplayTitle = (title: string) => {
    switch (title) {
      case 'HYUNDAI_ORIGINALS':
        return 'Hyundai Originals';
      case 'CHAMPION_BRANDS':
        return 'Champion Brands';
      case 'AFFILIATE':
        return '제휴카드';
      case 'MY_BUSINESS':
        return 'My Business';
      default:
        return title;
    }
  };
function DropDownTop({ title, onClick, isActive }: DropDownTopProps) {
```